2022/distributed-tracing.html

```diff
@@ -338,7 +328,8 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             </dt>
             <dd>The trace context standard relates to work around Server Timing done within the Web Performance group. Going forward
                 it will make sense to define how trace context headers - specifically response headers - are related to the server timing header.
-                The trace interchange format also can be used to represent server timing data.
+                The Working Group should coordinate with the Web Performance Working Group to ensure that the two specifications are
+                aligned. Server timing headers may be used to carry the trace context information as well as the baggage data.
```

Comment:
Maybe "may be used by applications" - to clarify this in not something the browser is expected to do on its own?